src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,74 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Calculator {
+
+    private static final String MATH_EXPRESSION_PATTERN = "^[+\\-]?\\d( ?[*\\-+/] ?\\d)*$";
+    protected Calculator(){}
+
+    public static Calculator of(){
+        return new Calculator();
+    }
+
+    public boolean isValidMathExpression(String rawExpression){
+        if(rawExpression == null || !rawExpression.matches(MATH_EXPRESSION_PATTERN)){
```

Comment:
코드 컨벤션 한번 신경써주시면 좋을 것 같습니다 🙇 
```suggestion
        if (rawExpression == null || !rawExpression.matches(MATH_EXPRESSION_PATTERN)) {
```